examples/rl/platoon/train/run.py

```diff
@@ -100,79 +120,95 @@ def main(args: argparse.Namespace):
         env.close()
 
 
-def run(
+def train(
     envs_train: Dict[str, gym.Env],
     envs_eval: Dict[str, gym.Env],
     config: Dict[str, Any],
+    agent_spec: AgentSpec,
 ):
+    print("\nStart training.\n")
+    scenarios_iter = cycle(config.scenarios)
+    save_dir = config.logdir / "train"
+    save_dir.mkdir(parents=True, exist_ok=True)
+    checkpoint_callback = CheckpointCallback(
+        save_freq=config.checkpoint_freq,
+        save_path=config.logdir / "checkpoint",
+        name_prefix="PPO",
+    )
 
-    if config.mode == "train":
-        print("\nStart training.\n")
-        scenarios_iter = cycle(config.scenarios)
-        model = sb3lib.PPO(
-            env=envs_train[next(scenarios_iter)],
-            tensorboard_log=config.logdir / "tensorboard",
-            verbose=1,
-            **network.combined_extractor(config),
-        )
+    for index in range(config.epochs):
+        scen = next(scenarios_iter)
```

Comment:
I found recently that it is possible to do the following
```suggestion
    scenarios_iter = islice(cycle(config.scenarios), config.epocs)
    for index, scen in enumerate(scenarios_iter):
```